components/script/dom/eventdispatcher.rs

```diff
@@ -103,10 +103,15 @@ fn dispatch_to_listeners(event: &Event, target: &EventTarget, event_path: &[&Eve
     }
 }
 
+pub enum EventStatus {
```

Comment:
You can derive `PartialEq` here so that you don't need `if let` to match for it.